bot/utils/utils.py

```diff
@@ -13,7 +13,7 @@ def is_root(ctx) -> bool:
 
 
 def is_valid(client, msg, expr="") -> bool:
-    return (
+    return bool(
```

Comment:
Is this _bool_ really necessary here?

Since your construct consists of "not ... and not ... and ..." I would assume any type checker recognizes it as a bool.

Reply:
> I would assume any type checker recognizes it as a bool.

Pylance with Basic type checking doesn't like it without bool